### PR TITLE
code example for update expanded text ads

### PR DESCRIPTION
**Pull request by xflesym**
add code example for update expanded text ads

### CODE REVIEW

File: examples/basic_operations/update_expanded_text_ads.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        ad_response.results[0].resource_name))
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description=('Updates an expanded text ad to the specified ad ID, '
```
Comment (wihl) [inline examples/basic_operations/update_expanded_text_ads.py]: Suggested reword: 'Updates the specified expanded text ad.'

File: examples/basic_operations/update_expanded_text_ads.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+from google.ads.google_ads.errors import GoogleAdsException
+from google.api_core import protobuf_helpers
+
+
+def main(client, customer_id, ad_id):
+    ad_service = client.get_service('AdService', version='v3')
+
+    ad_operation = client.get_type('AdOperation', version='v3')
+
+    # Create ad operation
```
Comment (wihl) [inline examples/basic_operations/update_expanded_text_ads.py]: s/Create/Update

File: examples/basic_operations/update_expanded_text_ads.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    ad.expanded_text_ad.headline_part2.value = 'Tickets on sale now'
+    final_url = ad.final_urls.add()
+    final_url.value = 'http://www.example.com'
+    final_mobile_url = ad.final_mobile_urls.add()
+    final_mobile_url.value = 'http://www.example.com/mobile'
+
+    fm = protobuf_helpers.field_mask(None, ad)
+    ad_operation.update_mask.CopyFrom(fm)
+
+    # Update the ad
```
Comment (wihl) [inline examples/basic_operations/update_expanded_text_ads.py]: Suggested reword: Updates the ad.

File: examples/basic_operations/update_expanded_text_ads.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        print(f'Request with ID "{ex.request_id}" failed with status '
+              f'"{ex.error.code().name}" and includes the following errors:')
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f'\t\tOn field: {field_path_element.field_name}')
+        sys.exit(1)
+
+    print('Ad with resource name {} was updated'.format(
```
Comment (wihl) [inline examples/basic_operations/update_expanded_text_ads.py]: Convert this to an f-string.  We are moving all our examples to take advantage of Python 3+ features.

**Comment by xflesym** [comment]
Addressed comments above.

File: examples/basic_operations/update_expanded_text_ads.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    ad.expanded_text_ad.headline_part2.value = 'Tickets on sale now'
+    final_url = ad.final_urls.add()
+    final_url.value = 'http://www.example.com'
+    final_mobile_url = ad.final_mobile_urls.add()
+    final_mobile_url.value = 'http://www.example.com/mobile'
+
+    fm = protobuf_helpers.field_mask(None, ad)
+    ad_operation.update_mask.CopyFrom(fm)
+
+    # Updates the ad
```
Comment (wihl) [inline examples/basic_operations/update_expanded_text_ads.py]: nit: add a '.'

File: examples/basic_operations/update_expanded_text_ads.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+from google.ads.google_ads.errors import GoogleAdsException
+from google.api_core import protobuf_helpers
+
+
+def main(client, customer_id, ad_id):
+    ad_service = client.get_service('AdService', version='v3')
+
+    ad_operation = client.get_type('AdOperation', version='v3')
+
+    # Update ad operation
```
Comment (fiboknacky) [inline examples/basic_operations/update_expanded_text_ads.py]: End with a period.

**Comment by xflesym** [comment]
Added punctuation in comments. Thanks David for the quick review!

File: examples/basic_operations/update_expanded_text_ads.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+        print(f'Request with ID "{ex.request_id}" failed with status '
+              f'"{ex.error.code().name}" and includes the following errors:')
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f'\t\tOn field: {field_path_element.field_name}')
+        sys.exit(1)
+
+    print(f'Ad with resource name {ad_response.results[0].resource_name} '
```
Comment (fiboknacky) [inline examples/basic_operations/update_expanded_text_ads.py]: Use a quote around the resource name too.

Comment (fiboknacky) [reply examples/basic_operations/update_expanded_text_ads.py]: And end the sentence with a period.

File: examples/basic_operations/update_expanded_text_ads.py (hunk 6 above)
Comment (xflesym) [reply examples/basic_operations/update_expanded_text_ads.py]: added.

**Comment by wihl** [comment]
> The file name must be update_expanded_text_ad.py

FYI, use `git mv oldfile newfile`

File: examples/basic_operations/update_expanded_text_ads.py (hunk 7 above)
Comment (xflesym) [reply examples/basic_operations/update_expanded_text_ads.py]: modified accordingly.

**Comment by xflesym** [comment]
filename changed.

**Pull request merged**